main.go

```diff
@@ -73,6 +73,7 @@ func saveDone(filename, title string) {
 
 func main() {
 	var indexPath string
+        var EcportedVar = 1
```

Comment:
**[gosimple]** <sub>reported by [reviewdog](https://github.com/haya14busa/reviewdog) :dog:</sub>
EcportedVar declared but not used